### PR TITLE
[platform-system-config] Explict chart values, velero, testing

### PR DESCRIPTION
**Pull request by npapapietro**

### CODE REVIEW

**Review by cameojokes — CHANGES_REQUESTED** [review]

File: charts/platform-system-config/values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    enabled: false
+    name: "cert-manager"
+    namespace: ""
+  istio:
+    enabled: false
+    name: "istio"
+    namespace: ""
+    mTLSMode: "STRICT"
+  velero:
+    enabled: false
```
[the remark this answers is not in the document]
Comment (npapapietro) [reply charts/platform-system-config/values.yaml:20]: I will in a followup pr

File: charts/platform-system-config/values.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  name: flux
+  namespace: flux-system
+
+components:
+  certManager:
+    enabled: false
+    name: "cert-manager"
+    namespace: ""
+  istio:
+    enabled: false
```
[the remark this answers is not in the document]
Comment (npapapietro) [reply charts/platform-system-config/values.yaml:15]: I will in a followup pr

File: charts/platform-system-config/values.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+prefix: "system"
+# Supported Values: `flux`
+# TODO: Support `argo`
+solution:
+  name: flux
+  namespace: flux-system
+
+components:
+  certManager:
+    enabled: false
```
[the remark this answers is not in the document]
Comment (npapapietro) [reply charts/platform-system-config/values.yaml:11]: I will in a followup pr

File: charts/platform-system-config/examples/values.test.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    enabled: true
+    name: "cert-manager"
+    namespace: "cert-manager"
+  istio:
+    enabled: true
+    name: "istio"
+    namespace: "istio"
+    mTLSMode: "STRICT"
+  velero:
+    enabled: false
```
[the remark this answers is not in the document]
Comment (npapapietro) [reply charts/platform-system-config/examples/values.test.yaml]: done

File: charts/platform-system-config/values.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  istio:
+    enabled: false
+    name: "istio"
+    namespace: ""
+    mTLSMode: "STRICT"
+  velero:
+    enabled: false
+    name: "velero"
+    namespace: ""
+    schedules:
```
[the remark this answers is not in the document]
Comment (npapapietro) [reply charts/platform-system-config/values.yaml:23]: It doesn't refer to ttl though, it refers to schedule CRs

**Review by cameojokes — APPROVED** [review]

**Pull request merged**